### DISCUSSION TITLE
Please check if the registration has been rejected

### DISCUSSION THREAD

**Discussion by wonhyunkwon** [WalletConnect/walletconnect-monorepo#1373] [Developer Support] [answered]
![스크린샷 2022-08-23 오전 9 17 03](https://user-images.githubusercontent.com/93938741/186041621-2445d3e1-0060-4c66-aaa1-74861ec86ff2.png)

I applied for registration a month ago and have been in pending review status until now.
The name of the wallet application is "Keywallet touch", and I will leave a video of how to use app store links and wallet connections.

I inquired about the same article on the discode channel, but there was no answer, so I am asking you again.

Please reply even if you are busy.

> App store
**https://apps.apple.com/kr/app/id1473941321**

> Video
I'm replacing the link with the one I posted on the discode channel because of the large capacity
**https://discord.com/channels/492410046307631105/1009645551630954506**

**Comment by hackyguru** [comment, marked as answer]
Thanks for being patient @name7777 
Reviewed your submission a few minutes ago.
